File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -759,6 +759,82 @@ document.Close()
+using (WordDocument document = new WordDocument())
+{
+        //Adds section to the document.
+        IWSection sec = document.AddSection();
+        //Adds paragraph to the section.
+        IWParagraph paragraph = sec.AddParagraph();
+        //Creates and Appends chart to the paragraph.
+        Chart chart = paragraph.AppendChart(446, 270);
+        //Sets chart type.
+        chart.ChartType = OfficeChartType.Pie;
+        //Sets chart title.
+        chart.ChartTitle = string.Empty;
+        //Sets data for chart.
+        chart.ChartData.SetValue(1, 1, "");
+        chart.ChartData.SetValue(1, 2, "Sales");
+        chart.ChartData.SetValue(2, 1, "Phyllis Lapin");
+        chart.ChartData.SetValue(2, 2, 141.396);
+        chart.ChartData.SetValue(3, 1, "Stanley Hudson");
+        chart.ChartData.SetValue(3, 2, 80.368);
+        //Creates a new chart series with the name “Sales”.
+        IOfficeChartSerie pieSeries = chart.Series.Add("Sales");
+        pieSeries.Values = chart.ChartData[2, 2, 3, 2];
+        //Sets category labels.
+        chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData[2, 1, 3, 1];
+        //Creates file stream.
+        using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../Result.docx"), FileMode.Create, FileAccess.ReadWrite))
+        {
+            //Saves the Word document to file stream.
+            document.Save(outputFileStream, FormatType.Docx);
+        }
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+'Adds section to the document.
+ Dim sec As IWSection =  document.AddSection() 
+'Adds paragraph to the section.
+ Dim paragraph As IWParagraph =  sec.AddParagraph() 
+'Creates and Appends chart to the paragraph.
+ Dim chart As Chart =  paragraph.AppendChart(446,270) 
+'Sets chart type.
+ chart.ChartType = OfficeChartType.Pie
+'Sets chart title.
+ chart.ChartTitle = String.Empty
+'Sets data for chart.
+ chart.ChartData.SetValue(1, 1, "")
+ chart.ChartData.SetValue(1, 2, "Sales")
+ chart.ChartData.SetValue(2, 1, "Phyllis Lapin")
+ chart.ChartData.SetValue(2, 2, 141.396)
+ chart.ChartData.SetValue(3, 1, "Stanley Hudson")
+ chart.ChartData.SetValue(3, 2, 80.368)
+'Creates a new chart series with the name “Sales”.
+ Dim pieSeries As IOfficeChartSerie =  chart.Series.Add("Sales") 
+ pieSeries.Values = chart.ChartData(2, 2, 3, 2)
+'Sets category labels.
+ chart.PrimaryCategoryAxis.CategoryLabels = chart.ChartData(2, 1, 3, 1)
+'Creates file stream.
+ Imports (FileStream outputFileStream = New FileStream(Path.GetFullPath("../../../Result.docx"), FileMode.Create, FileAccess.ReadWrite))
+ {
+     'Saves the Word document to file stream.
+      document.Save(outputFileStream, FormatType.Docx)
+ }
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Remove-chart-title/.NET%20Standard)
```

Comment:
Please use the link
https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Remove-chart-title

Reply:
Addressed the feedback